Bottom: 079dcf707d3682a2c1c39a7d46ae1ef94686edb0
Top:    9296380fe4b4ce17f2d03082d66e3b3e6dc2230b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:57 -0500

Remove unused pools


---
diff --git a/roles/bacula-director/templates/bacula-dir.conf.j2 b/roles/bacula-director/templates/bacula-dir.conf.j2
index db503866..c8e74344 100644
--- a/roles/bacula-director/templates/bacula-dir.conf.j2
+++ b/roles/bacula-director/templates/bacula-dir.conf.j2
@@ -259,16 +259,16 @@ Pool {
   Volume Retention = 365 days         # one year
 }
 
-# File Pool definition
-Pool {
-  Name = File
-  Pool Type = Backup
-  Recycle = yes                       # Bacula can automatically recycle Volumes
-  AutoPrune = yes                     # Prune expired volumes
-  Volume Retention = 365 days         # one year
-  Maximum Volume Bytes = 50G          # Limit Volume size to something reasonable
-  Maximum Volumes = 100               # Limit number of Volumes in Pool
-}
+# # File Pool definition
+# Pool {
+#   Name = File
+#   Pool Type = Backup
+#   Recycle = yes                       # Bacula can automatically recycle Volumes
+#   AutoPrune = yes                     # Prune expired volumes
+#   Volume Retention = 365 days         # one year
+#   Maximum Volume Bytes = 50G          # Limit Volume size to something reasonable
+#   Maximum Volumes = 100               # Limit number of Volumes in Pool
+# }
 
 
 # Scratch pool definition
@@ -278,13 +278,13 @@ Pool {
 }
 
 # LTO6 Test pool definition
-Pool {
-  Name = LTO6test
-  Pool Type = Backup
-  Recycle = yes                       # Bacula can automatically recycle Volumes
-  AutoPrune = yes                     # Prune expired volumes
-  Volume Retention = 365 days         # one year
-}
+# Pool {
+#   Name = LTO6test
+#   Pool Type = Backup
+#   Recycle = yes                       # Bacula can automatically recycle Volumes
+#   AutoPrune = yes                     # Prune expired volumes
+#   Volume Retention = 365 days         # one year
+# }
 
 #
 # Restricted console used by tray-monitor to get the status of the director
